src/client/utils/textKeys.test.tsx

```diff
@@ -1,8 +1,25 @@
-import { mount } from 'enzyme'
 import React from 'react'
-import { act } from 'react-dom/test-utils'
-import { nextTickAsync } from './misc'
-import { makeTextKeyResolver, TextKeyProvider, useTextKeys } from './textKeys'
+import { renderHook } from '@testing-library/react-hooks'
```

Comment:
Why not use the custom `renderHook` function from the `/test/utils.tsx` file here?

Reply:
It's just because we needed a different provider to wrap these tests. I wasn't sure if it's worth it to include the `TextKeyProvider` in the main `customRenderHook` function.

Just to not create abstractions too soon.

Reply:
I see! In a way I think it would be nice to always use the same `renderHook` function and have all the provider thingies there (to make it more consistent I guess, and be clear about that you're generally not supposed to import the `renderHook` directly from `'@testing-library/react-hooks'` and add the `StaticRouter` etc.), but maybe that's not a very AHA way to think 🙃

Reply:
I'll have a look into it and see if including `StaticRouter` across all tests will cause any issues!

Reply:
So the `TextKeyProvider` actually performs state updates in a `useEffect` -- so I think `StaticTextKeyProvider` is a better option for tests - I added that to the providers.

To test this particular logic, I've updated the test utils for that you can override the wrapper component on a per-test basis! What do you think of that solution?

Reply:
I like it! 👍 😀